### DISCUSSION TITLE
mPDF files for production environment (php_codesniffer?)

### DISCUSSION THREAD

**Discussion by vicentor** [mpdf/mpdf#1565] [Q&A] [answered]
Hello, first time here,
Is it safe to delete /vendor/php_codesniffer from mPDF files?

Our production environment security systems are freaking out over such files (including phpunit and tracy).

Thank you for any feedback!

**Comment by finwe** [comment, marked as answer]
Run composer install with --no-dev flag on your production environment. Problem solved.

Though if your security systems are freaking out over tracy, there is something wrong with your security systems.

**Reply by finwe** [reply]
Thinking about it again, dev dependencies should not be downloaded when you have mpdf as a dependency itself. So, to rephrase, run composer install with --no-dev flag wherever you add mPDF to your stack.

**Comment by vicentor** [comment]
Amazing, thank you for the follow-up and I agree with you!
